torch/csrc/generic/Tensor.cpp

```diff
@@ -816,6 +816,8 @@ PyTypeObject THPTensorStatelessType = {
 void THPTensor_(initCopyMethods)()
 {
   auto& h = THTensor_(copy_functions);
+  // copy from same type
+  THPInsertCopyFunction(h, &THTensor_(copy));
```

Comment:
Isn't this going to introduce duplicate entries? We're already registering the copies for every type.